jobs/JGDAS_ATMOS_ANALYSIS_WDQMS

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+set -x
+export PS4='$SECONDS + '
+date
+
+##############################
+# Source relevant config files
+##############################
+configs="base wdqms"
+export EXPDIR=${EXPDIR:-${HOMEgfs}/parm/config}
+for config in ${configs}; do
+  source "${EXPDIR}/config.${config}"
+  status=$?
+  (( status != 0 )) && exit "${status}"
+done
+
+###############################################################
+source "${BASE_ENV}/${machine}.env" wdqms
```

Comment:
Do we need an entry for this job in the env file(s)?

Reply:
Actually we don't need since there is no `APRUN` command being set.  All this is doing is setting the machine specific environment variables at the top of the `MACHINE.env` file.  An argument is required since it checks for it and aborts if not provided.